tgui/packages/tgui/interfaces/Fabrication/MaterialAccessBar.tsx

```diff
@@ -72,14 +72,16 @@ type MaterialCounterProps = {
 
 const MaterialCounter = (props: MaterialCounterProps, context) => {
   const { material, onEjectRequested } = props;
+  const { data } = useBackend<Material>(context);
+  const { SHEET_MATERIAL_AMOUNT } = data;
 
   const [hovering, setHovering] = useLocalState(
     context,
     `MaterialCounter__${material.name}`,
     false
   );
 
-  const canEject = material.amount > 2_000;
+  const canEject = material.amount > SHEET_MATERIAL_AMOUNT;
```

Comment:
`Material` has a `removable` boolean that can be used instead of `canEject`
's what i've done in #76170

https://github.com/tgstation/tgstation/blob/dd652b08ef5fe4f68e22daa5f09aa9f46927b90d/code/datums/components/material_container.dm#L547-L554

you can get rid of `canEject` and in L93 replace it with `material.removable`

Reply:
Actually it's faster to compute can_eject in the UI side rather than sending removable as extra data. Reduces bandwidth